src/tests/ftest/control/dmg_telemetry_io_latency.py

```diff
@@ -339,14 +340,186 @@ def test_io_latency_telmetry_metrics(self):
+                                        transfer_size):
+        """Verify IO dtx metrics after running IOR.
+
+        Args:
+            baseline_data (dict): dictionary of the initial io dtx "committed" metrics
+                                  before IOR run
+            metrics_data (dict): dictionary of io dtx "committed" metrics
+            test_metrics (dict): io dtx "committed" telemetry metrics
+            transfer_size(str): transfer_size used with ior
+
+        Returns:
+            status: (bool) True if metrics are verified for transfer size
+
+        """
+        block_size = self.params.get("block_size", "/run/*")
+        repetitions = self.params.get("repetitions", "/run/*")
+        test_ops = 3
+        status = True
+
+        metrics = {}
+        for test_metric in test_metrics:
+            metrics[test_metric] = 0
+            for host in self.hostlist_servers:
+                # test assumes one engine per host
+                for rank in self.server_managers[-1].get_host_ranks([host]):
+                    for target in range(self.server_managers[-1].get_config_value("targets")):
+                        value = metrics_data[test_metric][host][str(rank)][str(target)]["-"]
+                        metrics[test_metric] = metrics[test_metric] + value
+
+        # calculate diff from initial baseline test value for committed and max metrics
+        dtx_value = metrics["engine_io_dtx_committed"]
+        dtx_value -= baseline_data["engine_io_dtx_committed"]
+        max_value = metrics["engine_io_dtx_committed_max"]
+        max_value -= baseline_data["engine_io_dtx_committed_max"]
+        # expected min value after the test should be 1
+        min_value = metrics["engine_io_dtx_committed_min"]
+        # mean and std dev metrics only rely on the after test values
+        mean_value = metrics["engine_io_dtx_committed_mean"]
+        stddev_value = metrics["engine_io_dtx_committed_stddev"]
+
+        # DTX committed value should be equal to the number of transactions.
+        if transfer_size == "512":
+            transfer_size = "512B"
+
+        # num operations = (blocksize / xfersize) * repetitions + ops for create/remove
+        num_operations = convert_to_number(block_size) / convert_to_number(transfer_size)
+        num_operations *= repetitions
+        # test ops = (create dir + punch obj + punch dir) * repetitions
+        test_ops *= repetitions
+        num_operations += test_ops
+
+        if dtx_value != num_operations:
+            self.log.error("engine_io_dtx_committed NOT verified, %s != %s",
+                           dtx_value, num_operations)
+            status = False
+        if min_value != 1:
+            self.log.error("engine_io_dtx_committed_min != 1")
+            status = False
+        if not ((max_value >= dtx_value >= min_value) and (
+                max_value > mean_value > min_value) and (
+                    stddev_value < (max_value-min_value))):
```

Comment:
Some confused for the check of "mean_value" and "stddev_value", the "max_value" is the diff between current test and baseline, but "mean_value" and "stddev_value" are just for current test. It is meaningless to compare them directly. I would suggest to remove the check of "(max_value > mean_value > min_value) and (stddev_value < (max_value-min_value))", then the test will pass.